apps/system-e2e/src/lib/types.ts

```diff
@@ -35,6 +35,7 @@ enum BaseAuthority {
   ads = 'loftbru.dev01.devland.is',
   prod = 'island.is',
   local = 'localhost:4200',
+  applicationSystem = 'localhost:4242',
```

Comment:
I have added in the docs that we can (and should) run application system web on port 4200. Alternatively we need a better configuration support for local webs running on different ports.

Reply:
Should I revert this?